lib/src/types/c2_user.dart

```diff
@@ -1,29 +1,24 @@
 // ignore_for_file: invalid_annotation_target
 
-import 'package:c2logbook/src/types/c2_types.dart';
-import 'package:freezed_annotation/freezed_annotation.dart';
-
-part 'c2_user.freezed.dart';
-part 'c2_user.g.dart';
+part of 'index.dart';
 
 @freezed
 class C2User with _$C2User {
   const factory C2User(
       {@Default(0) int id,
       @Default('') String username,
-      @JsonKey(name: 'first_name') @Default('') String firstName,
-      @JsonKey(name: 'last_name') @Default('') String lastName,
+      @Default('') String firstName,
+      @Default('') String lastName,
       @Default('F') String gender,
-      @JsonKey(name: 'dob') @Default('1970-01-01') String birthday,
+      @Default('1970-01-01') String birthday,
       @Default('') String email,
       @Default('') String country,
-      @JsonKey(name: 'profile_image') String? profileImage,
```

Comment:
did i get a bunch of the JSON keys in this file wrong? or were you able to find a way to automate them based on the field name in dart?

Reply:
this can be put back, just a conflict. user isnt used